test/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -42,17 +58,22 @@ using ChainRules: level2partition, level3partition, chol_blocked_rev, chol_unblo
         @testset "the thing" begin
             X = generate_well_conditioned_matrix(rng, 10)
             V = generate_well_conditioned_matrix(rng, 10)
-            F, dX = rrule(cholesky, X)
+            F, dX_pullback = rrule(cholesky, X)
             for p in [:U, :L]
-                Y, (dF, dp) = rrule(getproperty, F, p)
-                @test dp isa ChainRules.DNERule
+                Y, dF_pullback = rrule(getproperty, F, p)
                 Ȳ = (p === :U ? UpperTriangular : LowerTriangular)(randn(rng, size(Y)))
+                (dself, dF, dp) = dF_pullback(Ȳ)
```

Comment:
why are there 3 returns here not 2?

Reply:
because get_property has 2 inputs, `F` and `p`